datanode/gateway/graphql/order_filter_resolver.go

```diff
@@ -0,0 +1,20 @@
+package gql
+
+import (
+	"context"
+
+	v2 "code.vegaprotocol.io/vega/protos/data-node/api/v2"
+	"code.vegaprotocol.io/vega/protos/vega"
+)
+
+type orderFilterResolver VegaResolverRoot
+
+func (o orderFilterResolver) Status(ctx context.Context, obj *v2.OrderFilter, data []vega.Order_Status) error {
+	obj.Statuses = data
+	return nil
+}
+
+func (o orderFilterResolver) TimeInForce(ctx context.Context, obj *v2.OrderFilter, data []vega.Order_TimeInForce) error {
+	obj.TimeInForces = data
+	return nil
+}
```

Comment:
this is a bit of an unusual pattern for resolvers, normally they return the thing right, rather than setting it somewhere? But perhaps it's fine in which case ignore me :)

Reply:
I'm not sure what you mean. `gqlgen` didn't generate any resolvers so I had to add these, but there's really no work to be done other than setting the OrderFilter property with the data provided so not sure what else there is to do. I think it's because there are already resolvers for the GraphQL enumerations to their gRPC counterparts.

Reply:
I mean, e.g. I was expecting to see here something like 
```
func (o orderFilterResolver) Status(ctx context.Context, obj *v2.OrderFilter, data []vega.Order_Status) Statuses, error {
	return obj.Statues, nil
}
```

But now I look upon it with fresh eyes maybe this is an _input_ resolver not an _output_ one and that is why it's setting not getting :)